src/PythonClient/src/requirements.txt

```diff
@@ -1 +1,2 @@
-pandas>=1.0.0,<2
\ No newline at end of file
+pandas>=1.0.0,<2
+confluent-kafka>=2.1.1,<2.2
```

Comment:
I see this added here but it is missing from the setup.py, not sure if intentional or not...don't really know the full development flow  yet =)